theme/templates/base.html

```diff
@@ -44,7 +44,7 @@
 <script src='{% static 'js/jquery-ui.js' %}'></script>
 <script src="{% static "js/bootstrap.3.0.2.min.js" %}"></script>
 <script src="{% static "js/bootstrap-extras.js" %}"></script>
-{% block extra_js %}{% endblock %}
+{#{% block extra_js %}{% endblock %}#}
```

Comment:
This block was causing the scripts to run twice. Do we really need to include this block tag here? After commenting it the extra_js block runs only once as it is supposed to.